server/lib/routes/get-verify-email.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const got = require('got');
+
+const logger = require('mozlog')('server.get-verify-email');
+const config = require('../configuration');
+
+module.exports = function () {
+
+  return {
+    method: 'get',
+    path: '/verify_email/',
+    process: function (req, res, next) {
+      req.url = '/';
+      console.log('items', req.query);
+
+      let data = {
+        code: req.query.code,
+        uid: req.query.uid
+      };
+
+      if (req.query.service) {
+        data.service = req.query.service;
+      }
+
+      if (req.query.reminder) {
+        data.reminder = req.query.reminder;
+      }
+
+      const fxaAccountUrl = config.get('fxaccount_url');
```

Comment:
Move this section above `+module.exports = function () {`.

This way you are not gonna be calling `config.get` on every request.